handbook/engineering/security/index.md

```diff
@@ -4,50 +4,39 @@ We think that security is an enabler for the business. Sourcegraph is committed
-  - Definition of success: We deploy a private repository containing encrypted secrets to Sourcegraph cloud, complete with a baseline for security monitoring. We are able to validate access to underlying hosts and the application in centralized logging.
+**Milestones:**
 
-**Host a capture the flag for Sourcegraph.com**
-
-  - Problem and rationale: Running a world class service capable of hosting private repositories is about more than application security. As examples, we need visibility into data loss, documentation for processes such as incident response, in addition to expanded defensive measures for both internal and external threat actors. These proxy for enhanced security concerns that a medium enterprise would require. This should be an ongoing, and escalating event, where we increase security controls and visibility whilst engaging on an ongoing capture the flag.
-  - Planned work:
-    - *Enhance policy documentation, and specific procedures such as incident response* - By its very nature, running a capture the flag generates ongoing events of interest, investigations, and incidents. Supporting this procedure both helps new hires get acclimated, and ensures the team is on the same page.
-    - *Deploy network visibility tools and integrate into our logging infrastructure* - A key part of attacking involves pivoting between hosts and service escalation. Through extending our visibility outbound, to the network layer, we would now have visibility into these attacks, and be able to analyze them. This include provides dashboards for network events of interest.
-    - *Conduct a mock capture the flag to tune security controls* - Given all of the logging, prior to announcing this event, we will test internally. This allows us to take the time to filter out various false positives from our logging, in addition to increasing or decreasing the verbosity of various logs, with the aim of increasing the value of security logging.
-    - *Encrypt stored data at rest* - possible future, TBD
-    - *Create data leakage alerts* - possible future, TBD
-  - Definition of success: We publicly commit to, and host an ongoing capture the flag, inviting the security community to participate, as we learn about our security measures. This will include both the repository from the previous goal, as well as a plain-text on-disk file containing the keys to unlock the repository.
-
-### Short term
-
-Our short term goals are documented in the current [tracking issue](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aissue+label%3Atracking+label%3Ateam%2Fsecurity+is%3Aopen).
+1. [Visibility into Sourcegraph Cloud's attack surface](#visibility-into-sourcegraph-clouds-attack-surface).
+   - I suspect we don't need to block on everything in that goal. Maybe we just need milestones 1-3? Let's discuss.
```

Comment:
I would suspect 1-4 would be sufficient to unblock, but 5 & 6 seem closely coupled with 4, so it might be reasonable to do all of them together. I know @chayim wants to focus on logging relatively soon.

Reply:
I think the key point is that we find and resolve all the easy/obvious things before we start asking people to attack us and rewarding large bounties.

It seems like "adding centralized logging" won't actually yield new insights unless we are under active attack, so that is why I think it isn't a pre-requisite to starting to award bounties (the next goal). Do you agree or disagree?

Reply:
I updated the PR and added the reasons there. If you disagree, please comment on the new diff.